mappings/net/minecraft/block/AbstractPlantBlock.mapping

```diff
@@ -1,8 +1,5 @@
-CLASS net/minecraft/unmapped/C_ymdmcjuw net/minecraft/block/AbstractPlantBlock
-	METHOD m_evmdilba copyState (Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_txtbiemp;)Lnet/minecraft/unmapped/C_txtbiemp;
-		ARG 1 from
-		ARG 2 to
-	METHOD m_pxnibpqz getStemHeadPos (Lnet/minecraft/unmapped/C_peaveboq;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_mmxmpdoq;)Ljava/util/Optional;
-		ARG 1 world
-		ARG 2 pos
-		ARG 3 block
+CLASS net/minecraft/unmapped/C_bgdokvym net/minecraft/block/AbstractPlantBlock
+	METHOD m_hjbkfomn canPlantOnTop (Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_peaveboq;Lnet/minecraft/unmapped/C_hynzadkk;)Z
```

Comment:
`canPlantAtop` would be the yarn core name here, I think I prefer yours

Reply:
oh I didn't change any methods, it's just showing that way because I renamed a class to an existing name